source/manual/sidekiq.html.md

```diff
@@ -8,33 +8,65 @@ type: learn
 ---
 
 Many of our applications use
-[Sidekiq](https://github.com/mperham/sidekiq) for background job processing.
+[Sidekiq](https://sidekiq.org) ([see repo](https://github.com/mperham/sidekiq))
+for background job processing.
 
-There's a [GOV.UK wrapper](https://github.com/alphagov/govuk_sidekiq) that will
-help you set it up.
+## Sidekiq on GOV.UK
+
+For redundancy, we have publishing apps running on multiple machines.
+We would have all sorts of race conditions and difficulties querying
+Sidekiq if each app on each machine had its own instance of Sidekiq.
+
+Therefore, we've build a GOV.UK wrapper for Sidekiq, called
+[govuk_sidekiq](https://github.com/alphagov/govuk_sidekiq). This
+allows all Sidekiq processes to talk to a single Redis instance. It
+also enables [request tracing](/manual/setting-up-request-tracing.html)
+and sends activity stats to Statsd, which forwards data to Graphite
+to be stored.
+
+## Retry logic
+
+Sidekiq has in built retry logic (turned on by default, but configurable).
+Successes, failures, job timings and retry counts are sent to `statsd`.
+
+Jobs do fail, this is not inherently bad and can happen for a number of
+reasons. When a job fails it gets [retried with an exponential backoff](https://github.com/mperham/sidekiq/wiki/Error-Handling#automatic-job-retry)
+(up to 21 days), as long as retries are enabled. A high number of retries
+signifies a bigger, less transient problem maybe occurring.
 
 ## Monitoring
 
-There are three approaches for monitoring Sidekiq, via the [Sidekiq Web interface](#sidekiq-web),
-or the [Grafana dashboard](#sidekiq-grafana-dashboard) or the [Console](#sidekiq-from-the-console).
+There are three approaches for monitoring Sidekiq:
+
+- via the [Sidekiq Web interface](#sidekiq-web-aka-sidekiq-monitoring)
+- or the [Grafana dashboard](#sidekiq-grafana-dashboard)
+- or the [Console](#sidekiq-from-the-console)
```

Comment:
```suggestion
- via the [Sidekiq Web interface](#sidekiq-web-aka-sidekiq-monitoring)
- via the [Grafana dashboard](#sidekiq-grafana-dashboard)
- via the [Console](#sidekiq-from-the-console)
```

Reply:
Personally I prefer what we have already, as it's clearer you're only expected to choose one of the options.

Reply:
If we say `There are three approaches for monitoring Sidekiq:`, it's clear that the list that follows is of those three approaches and that we're expected to choose one. I think the `or` works in a sentence (like it was before the change) but is unnecessary in a bullet point list.
Not a blocker though.

Reply:
Actually on second read, I agree! I read your comment from the 'landing page' of the PR, which only shows the suggestion, and not the sentences that precedes it. Now I prefer the `or`.

Have overwritten my commit 👍